Makefile

```diff
@@ -32,3 +32,6 @@ test-repl:
 hlint:
 	$(HLINT) --version
 	find src exec -name '*.hs' | xargs $(HLINT)
+
+ghci-no-type-errors:
+	cabal exec -- ghci $(FULL_SOURCES) -fdefer-type-errors
```

Comment:
we can add this if it makes your life easier, but what about adding it to ~/.ghci?  (that's what i am doing.)